### DISCUSSION TITLE
[Question]: how do i get the birdseye view like before

### DISCUSSION THREAD

**Discussion by jl-p** [blakeblackshear/frigate#11530] [Ask A Question]
### What is your question:

I installed 14 beta 1 
i always use a dedicated monitor where i have the birdseye view full screen.
when someone walk or drive up the driveway or the screen will show the camera or cameras. which make a blank screen drasticly changing and i see it from my peripheral vision and i see someone is walking to the door.
but how to do this in the new version.

And after installing and rebooting my machine i get a  404 error while accessing frigate. 
i can go to cameras or groups. or the other things. so thats no issue if i can access the fullscreen blank birdseye view again.
[
![frigate404](https://github.com/blakeblackshear/frigate/assets/10694270/244e7eee-52cf-4a06-9efd-bfd13b784944)
](url)

**Comment by banthungprong** [comment]
http://ip:8080/#birdseye

**Reply by jl-p** [reply]
cant find 8080 normaly it was 5000  but 8080 and 5000 dont answer

**Reply by NickM-27** [reply]
8080 is the authenticated port, 5000 is the unauthenticated port.

**Comment by jl-p** [comment]
i tried it but its not possible to open something 
ports 8080 and 5000 are not listening

![image](https://github.com/blakeblackshear/frigate/assets/10694270/45ff35a1-bab2-4afc-ae12-ed5e055ed837)

**Reply by NickM-27** [reply]
Then either the ports aren't mapped in docker or Frigate isn't running

**Comment by jl-p** [comment]
8080 still dont work. but changed 5000 to 5005 and now i have the birdseye back again
